LKMEX-whitelist/ElrondApes.md

```diff
@@ -0,0 +1,36 @@
+## Name of the project
+
+ElrondApes
+
+## Description of the project
+
+ElrondApes is a collection of 10,000 uniques NFTs on the MultiverseX blockchain.
+
+Website: https://elrondapes.com/
+
+## Purpose
+
+Our dApp (https://app.elrondapes.com/) currently provides a dashboard collection metrics, a community map and a staking feature. Basically, by staking your ElrondApes NFT into our contract, you're receiving LKMEX token as weekly rewards. These rewards come from a percentage of the sales royalties and the farming on the pool EGLD/USDC. We're internally discussing about changing the reward token, since the LKMEX will become non transferrable soon. We need to be whitelisted since stakers have not claimed their rewards from the contract yet.
+
+## Address of Smart Contract
+
+https://explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgq575mqgaqe4f53rqc5ssczkxtsx4fs26l3azq9zwkaw
+
+## Smart Contract Code
+
+Currently private repository
```

Comment:
We need access to the code.

Reply:
I just added you to the repo. Tell me if I need to add other collaborators

Reply:
https://github.com/EapesClub/elrond-apes-core-sc/blob/main/contracts/staking/src/rewards.rs